jetty-server/src/test/java/org/eclipse/jetty/server/LowResourcesMonitorTest.java

```diff
@@ -245,4 +249,41 @@ public void testMaxLowResourceTime() throws Exception
             }
         }
     }
+    
+    @Test
+    public void testStuckSelector() throws Exception
+    {
+        // TODO currently this is test needs human inspection of the log output
```

Comment:
@sbordet Not sure if we should keep this test, as it is has no asserts and needs visual confirmation.